packages/office-ui-fabric-react/src/components/MessageBar/examples/MessageBar.Styled.Example.tsx

```diff
@@ -1,222 +0,0 @@
-import * as React from 'react';
-import { MessageBarButton } from 'office-ui-fabric-react/lib/Button';
-import { Link } from 'office-ui-fabric-react/lib/Link';
-import { Label } from 'office-ui-fabric-react/lib/Label';
-import { Stack, StackItem } from 'office-ui-fabric-react/lib/Stack';
-import { MessageBar, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
-
-const log = (text: string): (() => void) => (): void => console.log(text);
-
-export const MessageBarStyledExample: React.StatelessComponent = () => (
```

Comment:
Might be worth preserving one of these as an example of how to do custom styling (just pick an accessible one)